app/controllers/articles_controller.rb

```diff
@@ -2,20 +2,26 @@ class ArticlesController < ApplicationController
   expose_decorated :article, -> { set_article }
   expose :comment, -> { set_comment }
   expose_decorated :company, -> { article.company }
+  expose :rating, -> { set_rating }
 
   layout "company"
 
   def show
     authorize article
   end
 
+  private
+
   def set_article
     Article.includes(:comments).find_by(id: params["id"])
```

Comment:
should we also includes ratings to avoid N+1?
```suggestion
    Article.includes(:comments, :ratings).find_by(id: params["id"])
```

Reply:
@mikhail-kilin sorry, I didn't notice. We can remove `set_article` method because `decent_exposure` gem already provide ready solution. You can just use: 
```
expose :article
expose :decorated_article, -> { article.decorate }
```